### DISCUSSION TITLE
Appium UiAutomator2 + WebDriverIO clearValue not working on a hybrid app

### DISCUSSION THREAD

**Discussion by SiNONiMiTY** [webdriverio/webdriverio#10371] [Q&A]
Title says, I am using JS to create tests. It went smoothly until I tried using the clearValue() method on an input field. It does not clear the pre-filled input.

Tried this approach for the said element
```
await inputField.click()
await inputField.clearValue()
await inputField.setValue( 'TEST' )
```

Any ideas or anything that I am missing.

```
WebDriverIO: 8.10.2
Appium: 2.0.0-beta.66
```

cross posted in Appium
https://github.com/appium/appium/discussions/18640

**Comment by christian-bromann** [comment]
These posts are better suited for the Appium project because it is the part that actually executes the click. I recommend to provide proper Appium logs as well as a reproducible example to make it possible to investigate.
